pkg/netns/netns_linux.go

```diff
@@ -127,7 +146,7 @@ func NewNS() (ns.NetNS, error) {
 		// Put this thread back to the orig ns, since it might get reused (pre go1.10)
 		defer func() {
 			if err := origNS.Set(); err != nil {
-				logrus.Errorf("unable to set namespace: %q", err)
+				logrus.Warnf("unable to set namespace: %q", err)
```

Comment:
Does this always trigger on rootless?

Reply:
As far as I have seen, since it is attempting to set it back to the golang thread at `/proc/...`